### DISCUSSION TITLE
Seeking Guidance on Learning AI and Accessing Public Sample Resources

### DISCUSSION THREAD

**Discussion by 0x131D491** [community/community#123361] [Programming Help]
I'm eager to delve into the world of artificial intelligence and machine learning, but I'm not sure where to begin. Can anyone recommend effective methods or resources for learning AI from scratch? Additionally, I'm keen on finding publicly available sample datasets or projects to practice with. Any suggestions or pointers would be greatly appreciated. Thank you in advance for your help!

**Comment by v08nike** [comment]
To start your journey in artificial intelligence and machine learning, begin by learning the basics of mathematics and programming, with a focus on Python. Explore online courses from platforms like Coursera and Udacity, along with textbooks such as "Artificial Intelligence: A Modern Approach." Engage in practical projects using platforms like Kaggle and GitHub to gain hands-on experience. Join online communities for discussions and seek guidance from forums like Reddit's r/MachineLearning. As you progress, consider specializing in areas like computer vision or natural language processing based on your interests. Stay curious, keep practicing, and enjoy the journey!

**Comment by goldieowner17** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @0x131D491! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. This question is more appropriate for Programming Help.

Also, you are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ebndev** [comment]
Hey @0x131D491!

You might be interested in joining GitHub's [#opensourcefriday AI and OSS](https://github.com/orgs/community/discussions/123787) chat happening this Friday, May 17th. 😄 Please feel free to leave your thoughts and questions in that discussion as well!
